plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -24,43 +25,37 @@ class CommandlineSpec:
 
     Attributes
     ----------
-    bounds: Optional[Tuple[float, float]]
+    bounds: Optional[Tuple[Any, Any]]
```

Comment:
CommandlineSpec should be deleted in my opinion. There is no need for it. You should convert from Override directly to the target.
See the the PR converting Ax for inspiration.